README.md

```diff
@@ -41,44 +49,59 @@ Azure also charges you for the number of transactions on managed disk. The calcu
 *As of 05/06/2018
 
 ## Usage
+
 ### I. Setup your local machine
+
 1. Sign up for a [Paid Azure subscription](https://azure.microsoft.com/en-us/pricing/purchase-options/). You need a paid subscription as the free account does not grant you access to GPU VMs.
-2. Sign up for an account on [zero tier VPN](https://www.zerotier.com/) and create a network. Make sure the network is set to **public**.
+2. Have Steam ready and logged in. You can specify client streaming options in Steam's Settings > Remote Play > Advanced Client Options. Make sure to limit the bandwidth of your local steam client to 15 or 30 Mbits (50 if you don't mind the extra data cost).
+
+You can also use Steam Link on a mobile device !
+
+#### I.B (Optional) Setup ZeroTier VPN
+
+For some scenarios other than Steam Remote Play
+
+3. Sign up for an account on [zero tier VPN](https://www.zerotier.com/) and create a network. Make sure the network is set to **public**.
 Note down the network id.
-3. Download and install zero tier VPN on your local machine. Join the network using the network ID noted in the previous step. **Make sure your local machine connect to the network BEFORE the VM does!**
+4.  Download and install zero tier VPN on your local machine. Join the network using the network ID noted in the previous step. **Make sure your local machine connect to the network BEFORE the VM does!**
 
 ### II. Automatically Deploy Your Azure VM
+
 #### Automated Standard
+
 <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fecalder6%2Fazure-gaming%2Fmaster%2FStandard.json" target="_blank">
     <img src="http://azuredeploy.net/deploybutton.png"/>
 </a>
 
 #### Automated Low Priority
+
 <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fecalder6%2Fazure-gaming%2Fmaster%2FLowPri.json" target="_blank">
     <img src="http://azuredeploy.net/deploybutton.png"/>
 </a>
 
 Click on the button above for your desired VM type and fill out the form. You'll need to fill in:
+
 * Subscription: your paid subscription
 * Resource group: create a new one and name it anything you like
-* Location: pick the location closest to you. Note that not every location has the VM with M60 graphics card. Check [this website](https://azure.microsoft.com/en-us/global-infrastructure/services/) for whether a region supports NV6 VM.
-* Admin username and password: the login credentials for the local user.
+* Location: pick the location closest to you. Note that not every location has the VM with M60 graphics card. Check [this website](https://azure.microsoft.com/en-us/global-infrastructure/services/) for whether a region supports NV6 VM.* Admin username and password: the login credentials for the local user.
```

Comment:
Nit: newline for Admin username and password.

Reply:
You're right, not sure how I merged these lines. 
```suggestion
* Location: pick the location closest to you. Note that not every location has the VM with M60 graphics card. Check [this website](https://azure.microsoft.com/en-us/global-infrastructure/services/) for whether a region supports NV6 VM.
* Admin username and password: the login credentials for the local user.
```